workflow/rules/common.smk

```diff
@@ -486,26 +484,14 @@ def get_report_batch(wildcards):
 def get_merge_calls_input(ext="bcf"):
     def inner(wildcards):
         return expand(
-            "results/calls/{{group}}.{vartype}.{{event}}.{filter}.fdr-controlled.{ext}",
+            "results/calls/{{group}}.{vartype}.{{event}}.fdr-controlled.{ext}",
             ext=ext,
             vartype=["SNV", "INS", "DEL", "MNV", "BND", "INV", "DUP", "REP"],
-            filter=config["calling"]["fdr-control"]["events"][wildcards.event][
-                "filter"
-            ],
         )
 
     return inner
 
 
-def get_merge_calls_input_report(wildcards, ext="bcf"):
```

Comment:
I removed this as this seems to be dead code. At least I could not find where this function is called.